internal/condition/metrics/metrics.go

```diff
@@ -23,15 +23,13 @@ func init() {
 	m := &ConditionMetrics{
 		compilationTime: promauto.NewHistogram(prometheus.HistogramOpts{
 			Namespace: build.ProjectName,
-			Subsystem: subsystem,
```

Comment:
Removed for consistency with the other metrics. This also adds a `subsystem_` prefix to the FQN.